py-polars/tests/unit/namespaces/test_strptime.py

```diff
@@ -669,6 +669,20 @@ def test_to_datetime_use_earliest(exact: bool) -> None:
         ).item()
 
 
+def test_to_datetime_naive_format_and_time_zone() -> None:
```

Comment:
unrelated to this PR, but codecov flagged it as uncovered, so it's a good chance to add it